flink-cdc-cli/src/main/java/org/apache/flink/cdc/cli/CliExecutor.java

```diff
@@ -69,10 +69,10 @@ public PipelineExecution.ExecutionInfo run() throws Exception {
         return execution.execute();
     }
 
-    private PipelineComposer getComposer(Configuration flinkConfig) {
+    private PipelineComposer getComposer(Configuration flinkConfig, Configuration pipelineConfig) {
```

Comment:
Can we set flink config in PipelineComposer.compose https://github.com/apache/flink-cdc/blob/1c69a650e3187734962e824e40ed35de0d94ccfc/flink-cdc-composer/src/main/java/org/apache/flink/cdc/composer/flink/FlinkPipelineComposer.java#L96 method to avoid modifying these constructor?